src/partitioner/histogram.py

```diff
@@ -0,0 +1,39 @@
+"""Utilities for generating and manipulating object count histograms"""
+
+import healpy as hp
+import numpy as np
+
+from partitioner.io_utils import read_dataframe
+
+
+def empty_histogram(highest_order):
+    """Use numpy to create an histogram array with the right shape, filled with zeros"""
+    return np.zeros(hp.order2npix(highest_order), dtype=np.ulonglong)
+
+
+def generate_partial_histogram(
+    file_path,
+    highest_order,
+    file_format,
+    ra_column,
+    dec_column,
+):
+    """Generate a histogram of counts for objects found in the indicated file_path"""
```

Comment:
Here and throughout the code I would suggest the doc strings include the input arguments information so that this can be picked up by document generation in the future.

Reply:
Updated all the docstrings. Thanks for the pointer!